eox_nelp/course_experience/api/v1/views.py

```diff
@@ -0,0 +1,192 @@
+"""The generic views for course-experience API. Nelp flavour."""
```

Comment:
it'd be great if you add example to every view since it's hard to know which url should be used in every case

Reply:
[04d8553](https://github.com/eduNEXT/eox-nelp/pull/34/commits/04d85533bd4e2cdb30793cab482c4494160f16e3)